var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
```

Comment:
Meh. I don't put much faith in maintaining previous behavior if we know there's a better way to do something. Before #3691 we were building `+wide` but disabling the lib suffixes. Because of this, most of the packages are tuned to work with `-lncurses`. I guess the variant doesn't matter that much as long as most packages can be built with either.